lib/shelley/src/Cardano/Wallet/Shelley.hs

```diff
@@ -354,21 +352,20 @@ serveWallet
                 poolsDbTracer
                 (Pool.defaultFilePath <$> dir)
                 (timeInterpreter nl)
-                $ \db -> do
-            let spl = newStakePoolLayer nl db
+                $ \db@DBLayer{..} -> do
+
+            forM_ settings $ atomically . putSettings
             void $ forkFinally (monitorStakePools tr gp nl db) onExit
-            fetch <- fetchFromRemote trFetch fetchStrategies
-                <$> newManager defaultManagerSettings
-            void $ forkFinally (monitorMetadata tr gp fetch db) onExit
+            spl <- newStakePoolLayer nl db $ forkFinally' (monitorMetadata tr gp db) onExit
```

Comment:
`onExit` here will print an error in the logs every time the worker exits saying that it has exited unexpectedly. That is **not** what we want. 
Logging something is a good idea, but it should not be an error and not suggests that something unexpected happened.